packages/core/src/page.js

```diff
@@ -174,7 +175,7 @@ export class Page {
     }
 
     // wait for any final network activity before capturing the dom snapshot
-    await this.network.idle();
+    await this.network.idle(undefined, undefined, captureResponsiveAssetsEnabled);
```

Comment:
wait why are we doing this ? its basically not doing anything at all and not waiting for page idle